crates/secrets-types/src/lib.rs

```diff
@@ -34,13 +34,21 @@ pub struct Context {
     pub host_jwt: String,
     /// The application the entity belongs to.
     /// TODO: should this also be a JWT, but signed by the host?
-    pub application: Option<Application>,
+    pub application: Application,
 }
 
 /// The application that the entity belongs to.
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, Default)]
 pub struct Application {
+    /// The name of the application.
+    #[serde(default)]
     pub name: String,
```

Comment:
I was going to leave a comment about making this optional again, but as long as we're okay with thinking that an empty app name == no application this works fine. It feels weird to make `name` optional but `policy` not